src/pages/EditMonitor.vue

```diff
@@ -866,7 +887,7 @@ message HealthCheckResponse {
                 this.monitor.headers = JSON.stringify(JSON.parse(this.monitor.headers), null, 4);
             }
 
-            if (this.isAdd) {
+            if (this.isAdd || this.isClone) {
```

Comment:
I would split it into two conditions so that it would be easier to make changes to either case in the future

Reply:
And in general I would suggest a redo on the switchcases, it's not clear or obvious that the logic for editMonitor is in the other branch until you read the whole thing.

Reply:
I would disagree. There is a lot of things that could be improved in this repo, but duplicating code does not make it better

Reply:
Please make another PR for making it more clear that `editMonitor` is in the else bracket. Unrelated to this PR